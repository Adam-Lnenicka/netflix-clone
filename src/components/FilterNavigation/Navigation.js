import React from "react";
import MovieSortCriteria from "../MovieSortCriteria";
import FilterNavigation from "./FilterNavigation";

const Navigation = () => {
  return (
    <div
      className="flex navigation"
      style={{ justifyContent: "space-between" }}
    >
      <FilterNavigation />
      <div className="flex">
        <p>SELECT CRITERIA</p>
        <MovieSortCriteria />
      </div>
    </div>
  );
};

export default Navigation;
