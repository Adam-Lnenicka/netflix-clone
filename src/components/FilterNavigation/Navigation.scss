.navigation {
  max-width: 1300px;
  padding: 2em 4em;
  padding-bottom: 0;
  margin: 0 auto;
  justify-content: space-between;
}

.filter-navigation {
  border-bottom: 2px #555555 solid;
  button {
    color: white;
    background-color: #232323;
    text-transform: uppercase;
    border: 3px solid #232323;
    cursor: pointer;
    &:hover {
      border-bottom: 3px solid #f65261;
    }
  }
}

.sort-movies {
  background-color: #232323;
  color: white;
  padding: 1rem;
  text-transform: uppercase;
  option {
    background-color: #232323;
    text-transform: uppercase;
  }
}
