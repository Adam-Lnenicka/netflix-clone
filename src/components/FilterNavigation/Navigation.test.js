import React from "react";
import Navigation from "./Navigation";
import toJSON from "enzyme-to-json";
import { shallow } from "enzyme";

describe("Navigation", () => {
  it("Matches navigation snapshot", () => {
    const wrapper = shallow(<Navigation />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
