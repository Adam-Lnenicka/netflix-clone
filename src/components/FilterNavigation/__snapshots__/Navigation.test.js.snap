// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Matches navigation snapshot 1`] = `
<div
  className="flex navigation"
>
  <FilterNavigation />
  <div
    className="flex"
  >
    <p>
      SELECT CRITERIA
    </p>
    <MovieSortCriteria />
  </div>
</div>
`;

exports[`snapshot 1`] = `
<div
  className="flex navigation"
>
  <FilterNavigation />
  <div
    className="flex"
  >
    <p>
      SELECT CRITERIA
    </p>
    <MovieSortCriteria />
  </div>
</div>
`;
